### PR TITLE
Gdr 2371

### PR DESCRIPTION
**Pull request by gladkia**
See: https://github.com/gdrplatform/gDRstyle/pull/71
Preview: https://gdrplatform.github.io/gDRtestData/

### CODE REVIEW

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    branches: [main, master, test_ci_pkgdown]
+
+jobs:
+  trigger_pkgdown:
+    uses: gdrplatform/gDRstyle/.github/workflows/pkgdown.yaml@GDR-2371_2
+    with:
+      USERNAME: ${{ github.actor }}
+      PACKAGE_NAME: ${{ github.event.repository.name }}
+      BRANCH_NAME: ${{ github.head_ref }}
+      DEFAULT_BRANCH: master
```
Comment (gladkia) [inline .github/workflows/pkgdown.yaml:22]: @marcin-kam if we remove DEFAULT_BRANCH from the pipeline in gDRstyle - I will also remove it from here.

Comment (marcin-kam) [reply .github/workflows/pkgdown.yaml:22]: 👍

File: .github/workflows/pkgdown.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+name: pkgdown
+
+on:
+  push:
+    branches: [main, master, test_ci_pkgdown]
+
+jobs:
+  trigger_pkgdown:
+    uses: gdrplatform/gDRstyle/.github/workflows/pkgdown.yaml@GDR-2371_2
```
Comment (marcin-kam) [inline .github/workflows/pkgdown.yaml:17]: Don't forget to change branch

**Review by marcin-kam — APPROVED** [review]

File: _pkgdown.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+url: https://gdrplatform.github.io/gDRutils/
```
Comment (j-smola) [inline _pkgdown.yml:1]:
```suggestion
url: https://gdrplatform.github.io/gDRtestData/
```

**Review by j-smola — COMMENTED** [review]
Shouldn't the version of packages be updated? 
gDRstyle (>= 1.1.3)
RoxygenNote: 7.3.0

**Comment by gladkia** [comment]
> Shouldn't the version of packages be updated? gDRstyle (>= 1.1.3) RoxygenNote: 7.3.0

Done.

**Pull request merged**